tools/archive/model.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""WebCompat Issue Model for archives."""
+
+from dataclasses import dataclass
```

Comment:
Pretty cool, https://docs.python.org/3/library/dataclasses.html -- I hadn't learned about this yet.

Reply:
@miketaylr you may want to read this too. 
* https://realpython.com/python-data-classes/
* https://www.python.org/dev/peps/pep-0557/

Reply:
Thanks, the realpython article is very nice.